### PR TITLE
Add Travis CI Integration

### PR DESCRIPTION
**Pull request by sagemlee**
-Create Travis.yml
-Connect Heroku to Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+before_install:
```
Comment (EricLarson2020) [inline .travis.yml:1]: good work setting this up

Comment (EricLarson2020) [reply .travis.yml:1]: we probably will need to store the secure key in application.yml

Comment (jpc20) [reply .travis.yml:1]: Nice! 
Doesn't Travis store the key for us and the key in the travis.yml file is a secure one?

Comment (sagemlee) [reply .travis.yml:1]: Yup, please ignore my previous comment, Stella is right, this is encrypted.

**Pull request merged**